invenio_rdm_migrator/state.py

```diff
@@ -213,7 +213,7 @@ def _initialize_db(cls, metadata):
             sa.Column("id", sa.Integer, unique=True, nullable=False),
             sa.Column("pid_type", sa.String, nullable=False),
             sa.Column("status", sa.String, nullable=False),
-            sa.Column("created", sa.DateTime, nullable=False),
+            sa.Column("created", sa.String, nullable=False),
```

Comment:
Is this an issue with SQLite or is there a different value stored in the field?

Reply:
We need `datetime` objects, I think I will revert this back now that we need a transform mixin for dates